examples/systemcalls/systemcalls.c

```diff
@@ -16,8 +34,14 @@ bool do_system(const char *cmd)
  *   and return a boolean true if the system() call completed with success
  *   or false() if it returned a failure
 */
-
-    return true;
+    //int system(const char *cmd) - used to do fork(), exec() and wait() 	call in a single statement.
+    int sys_call = system(cmd);
+    if(sys_call == -1){
```

Comment:
@srki3050 is `-1` the right return code to check here?  Does this cover all the cases in the description?